Gruntfile.js

```diff
@@ -51,13 +51,14 @@ module.exports = function(grunt) {
     }
   };
 
-  var prefixMatch = "\\n *(function |var |static )?";
+  var regexLF = (grunt.util.linefeed === "\r\n" ? "(\\u000D\\n)" : "\\n"); //get the correct Lf for the operating system
```

Comment:
Just curious.  if we take the `escapeRegExp` function from here https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex , does that work here (Please reference the stackoverflow post if you do use it)?

Otherwise if this does not work, please put a comment above this line that details what you are doing here.